proxy/outprocess/src/module_loaders/outprocess_module.c

```diff
@@ -395,6 +395,8 @@ int outprocessControlThread(void *param)
 				}
 				else
 				{
+                    /*Codes_SRS_OUTPROCESS_MODULE_24_061: [ Once the control channel has been restarted and Create Message was sent, it shall send a Start Message to the module host. ]*/
+                    send_start_message(handleData);
```

Comment:
outprocessControlThread is created in Outprocess_Start, that means that the gateway was started when it can reach this code to recreate the out of process module.